hpi-archetype/pom.xml

```diff
@@ -15,15 +14,34 @@
   <!-- $ --><version>1.0-SNAPSHOT</version><!-- /$ -->
   <packaging>hpi</packaging>
 
+  <properties>
+    <!-- Baseline Jenkins version you use to build and test the plugin. Users must have this version or newer to run. -->
+    <jenkins.version>1.580.1</jenkins.version>
+    <!-- Java Level to use. Default is 7, but was only required by Jenkins >= 1.614. -->
+    <java.level>6</java.level>
+    <!-- Other properties you may want to use:
+         ~ jenkins-test-harness.version: The JTH version used to test plugin. The default is ${jenkins.version}.
+         ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin. Current default is 1.115.
+         ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin. Current default is 1.17
+         ~ java.level.test: The Java version to use to build the plugin tests. Current default is 7.
```

Comment:
no I mean that it's default value should be `${java.level}` not 7.

What you have described in the comments is that the main code is Java 6 and the tests have Java 7... this is of course on the one hand handy as the tests can then use try with resources and multi-catch... but now you cannot run the tests using Java 6 so your Java 6 code coverage is 0% and you have to hope that there is no subtle JVM version bug introduced.

Internally - at cloudbees - a person whose surname rhymes with stick changed one of our plugins to have the tests compiled with Java 8 because the IDE they used to be a developer for happens to support different Java levels for tests from production code... making it very hard for other people to work on that plugin... and making it impossible to test that plugin with Java 7 (or even Java 6 at the time the change was made)... The experience of working with that code base leads me to strongly recommend having the test level almost always the same as the main level